source/lib-intro.tex

```diff
@@ -1592,15 +1592,15 @@
 \begin{codeblock}
 #include <utility>
 
-// Requires: \tcode{std::forward<T>(t)} shall be swappable with \tcode{std::forward<U>(u)}.
+// \requires \tcode{std::forward<T>(t)} shall be swappable with \tcode{std::forward<U>(u)}.
```

Comment:
I wouldn't change these two comments. They're in an example, not in normative wording, and the status quo seems just fine.